source/api/image/3.0/index.md

```diff
@@ -518,21 +518,22 @@ The request for the image information _MUST_ conform to the URI template:
 ```
 {: .urltemplate}
 
-The syntax for the response is [JSON-LD][org-w3c-json-ld]. The HTTP `Content-Type` header of the response _SHOULD_ have the value `application/ld+json` with the `profile` parameter given as the context document: `http://iiif.io/api/image/3/context.json`.
+The syntax for the response is [JSON-LD][org-w3c-json-ld]. If the server receives a request with an `Accept` header, it _SHOULD_ respond following the rules of [content negotiation][org-rfc-7231-conneg]. Note that content negotiation requests _MAY_ include `profile` and other parameters with any content type. 
```

Comment:
Note that the content type provided in the `Accept` header of the request _MAY_ include parameters, for example `profile` or `charset`.  

?

Reply:
Agreed, with content types (plural)